File: 参考/comp9041-Q/cs9041/lab05/echon.pl
#!/usr/bin/perl -w

if (@ARGV != 2){
	print "Usage: $0 <number of lines> <string>","\n";
	exit 1;
}


$num = $ARGV[0];
$line = $ARGV[1];

if (not $num =~ m/^\d+$/){
	print "$0: argument 1 must be a non-negative integer","\n";
	exit 1;
}
$i = 0;
while ($i < $num) {
	print $line,"\n";
	$i++;
}

File: 参考/comp9041-454/9041-labonserver/lab06/echon.py
#!/usr/bin/python
from sys import *
from re import *
if len(argv)!=3:
    print("Usage: ./echon.py <number of lines> <string>")
    exit()
try:
    num=int(argv[1])
except:
    print("./echon.py: argument 1 must be a non-negative integer")
    exit()
if num>=0:
    for i in range(0,num):
        print(argv[2])
else:
    print("./echon.py: argument 1 must be a non-negative integer")
    exit()
